app/views/announcements/index.html.erb

```diff
@@ -1,6 +1,40 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Announcements</h1>
+<h1 style="padding-left=20px">Announcements</h1>
+
+<div class="cardContainers">
+    <% @announcements.each do |announcement| %>
+        <div class="announcementCard">
+          <div class="announcementCardTitle">
+            <%= announcement.title %>
+          </div>
+
+          <div class="announcementCardBody">
+              <div class="announcementCardBody1">
+                  <div>
+                    <Strong>Description: </Strong>
+                          <%= announcement.title %>
+                  </div>
+
+              </div>
+
+              <div class="announcementCardBody1">
+                  <div>
+                    <Strong>Description: </Strong>
```

Comment:
What do the two different descriptions on the card represent?

Reply:
It is not completely done yet. I was going to wait until the events functionality is done, since it is going to be on the same page. This way I dont have to fix it twice.  What do you think?

Reply:
Sounds good to me!